README.md

```diff
@@ -304,6 +304,7 @@ _MDAs - IMAP/POP3 software_
 _Mailing lists servers and mass mailing software - one message to many recipients._
 
   * [Dada Mail](http://dadamailproject.com/) - A web-based list management system that can be used for announcement lists and/or discussion lists. ([Source Code](https://github.com/justingit/dada-mail)) `GPLv2` `Perl`
+  * [Mail For Good](https://github.com/freeCodeCamp/mail-for-good) `⚠`  - An open source email campaign management tool for nonprofits.  `BSD` `Javascript`
```

Comment:
> Just added the ⚠ tag to Mail for Good addition. All good now?

All right, let's do that.

Syntax errors:

 * double space after `⚠`
 * double space before license